### ISSUE TITLE
cli broken in 4.0.0

### ISSUE THREAD

**Issue by denouche** [markedjs/marked#2268]
**Marked version:**
v4.0.0

**Nodejs version:**
v16.13.0

**Describe the bug**
I'm running a fresh Ubunutu 21.10, and I have this behavior:
```
$ marked -h
man: command exited with status 1: sed -e '/^[[:space:]]*$/{ N; /^[[:space:]]*\n[[:space:]]*$/D; }' | LESS=-ix8RmPm Manual page marked\.1 ?ltline %lt?L/%L.:byte %bB?s/%s..?e (END):?pB %pB\%.. (press h for help or q to quit)$PM Manual page marked\.1 ?ltline %lt?L/%L.:byte %bB?s/%s..?e (END):?pB %pB\%.. (press h for help or q to quit)$ MAN_PN=marked\.1 pager

```

**To Reproduce**

```
$ npm install -g marked@v4.0.0
$ marked -h
```

**Comment by UziTech** [comment]
this actually might be a bug in ubuntu. It works perfectly fine in 20.04

**Comment by filtermap** [comment]
And when I run this command, Marked outputs raw markdown file (it should be a html file).

```
marked ./something.md -o ./something.html
```

**Comment by UziTech** [comment]
@filtermap those issues should be fixed in #2294
